instrumentation/apache-httpclient/apache-httpclient-5.2/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v5_2/internal/Experimental.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apachehttpclient.v5_2.internal;
+
+import static java.util.logging.Level.FINE;
+
+import io.opentelemetry.instrumentation.apachehttpclient.v5_2.ApacheHttpClientTelemetryBuilder;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+
+/**
+ * This class is internal and experimental. Its APIs are unstable and can change at any time. Its
+ * APIs (or a version of them) may be promoted to the public stable API in the future, but no
+ * guarantees are made.
+ */
+public class Experimental {
```

Comment:
Sorry, I don't fully understand why we need to create a `Experimental` class here. If it's an experimental class, I guest the name may be changed at least in the future. If we recommend users to use it instead of original method with @Deprecated. Users may need to make a change when we change the `Experimental` class. Is there any discussion context about this point before?

Reply:
our stability guarantee doesn't apply to things under `.internal.` packages

https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md#compatibility-requirements

> The public API is any public class or method that is not in a package which includes the word `internal`.

I may not have understood your question though, let me know if this answers or if it was something else, thanks

Reply:
> The public API is any public class or method that is not in a package which includes the word `internal`.

In fact, I want to know why we use an `Experimental` class to wrap the previous `setEmitExperimentalHttpClientMetrics` method. 
Thank for your hints, because we need to stabilize the HTTP library here, for some experimental methods, in order to modify them easily, we put them in `internal` package firstly. Is it correct?

Reply:
yep, the experimental methods won't live forever, so we don't want to make them part of public API